ha/virsh/tests/resource_iscsi_target_test.sh

```diff
@@ -41,25 +46,25 @@ test_iscsi_target_resources_are_started() {
 
   # Wait for changes to be applied and check if the resource are started
   sleep 5
-  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo crm status")
+  cluster_status=$(run_command_in_node "${IP_VM01}" "sudo pcs status")
   echo "${cluster_status}" | grep "${TARGET_RESOURCE_NAME}" | grep Started
   assertTrue "Could not infer ${TARGET_RESOURCE_NAME} readiness from status output:\n${cluster_status}" $?
   echo "${cluster_status}" | grep "${LUN_RESOURCE_NAME}" | grep Started
   assertTrue "Could not infer ${LUN_RESOURCE_NAME} readiness from status output:\n${cluster_status}" $?
 
   # Check if both resource are running on the same node
   target_location=$(run_command_in_node "${IP_VM01}" \
-			"sudo crm resource locate ${TARGET_RESOURCE_NAME} | cut -d ':' -f2")
+			"sudo pcs status | grep ${TARGET_RESOURCE_NAME} | rev | cut -d ' ' -f1 | rev")
```

Comment:
That's quite the transformation to get the last column :)

I have one suggestion to make that parsing more obvious, and another to use a more specific `pcs` command. Here is the build-up to the final result:

```
# pcs resource status iscsi-target-rsc 
  * Resource Group: iscsi-target-group:
    * iscsi-target-rsc  (ocf:heartbeat:iSCSITarget):     Started ha-agent-virsh-jammy-resource-iscsi-target-node02
```
Unfortunately another `grep` is needed, so it's not much different than parsing the full output of `pcs status`:
```
# pcs resource status iscsi-target-rsc  | grep iscsi-target-rsc
    * iscsi-target-rsc  (ocf:heartbeat:iSCSITarget):     Started ha-agent-virsh-jammy-resource-iscsi-target-node02
```
And now:
```
# pcs resource status iscsi-target-rsc  | grep iscsi-target-rsc | sed -r 's,.*Started (.*)$,\1,'
ha-agent-virsh-jammy-resource-iscsi-target-node02
```

Another alternative is `awk`:
```
# pcs resource status iscsi-target-rsc  | grep iscsi-target-rsc | awk '{print $5}'
ha-agent-virsh-jammy-resource-iscsi-target-node02
```
In the end, given the lack of output formatting options of `pcs`, and the lack of a `locate` command like `crm`'s, it's a matter of preference. And if you still prefer your double-rev, it's fine. But I kind of prefer to at least gate on the `Started` that we see in the output. That can be added to the initial `grep` perhaps. So you could keep your doble-rev, but add something like `.*Started` to the end of the `grep`.

Reply:
At the end of this diff, I see that `get_name_node_running_resource` does the same parsing, but stores the result in a global variable instead of, say echo it to stdout. Was just wondering if we could put this all together in one function. But maybe for another branch if you prefer.